sympy/matrices/tests/test_solvers.py

```diff
@@ -526,6 +526,31 @@ def test_issue_19815():
                           [w['tau5']],
                           [1]])
 
+    from sympy.abc import j,f
+    # https://github.com/sympy/sympy/issues/20046
+    A = Matrix([
+    [1,  1, 1,  1, 1,  1, 1,  1,  1],
+    [0, -1, 0, -1, 0, -1, 0, -1, -j],
+    [0,  0, 0,  0, 1,  1, 1,  1,  f]
+    ])
+
+    sol_1=Matrix(list(linsolve(A))[0])
+
+    tau0, tau1, tau2, tau3, tau4 = symbols('tau:5')
+
+    assert sol_1 == Matrix([[-f - j - tau0 + tau2 + tau4 + 1],
+                          [j - tau1 - tau2 - tau4],
+                          [tau0],
+                          [tau1],
+                          [f - tau2 - tau3 - tau4],
+                          [tau2],
+                          [tau3],
+                          [tau4]])
+
+    # https://github.com/sympy/sympy/issues/19885
+    sol_2 = A[ : , : -1 ] * sol_1 - A[ : , -1 ]
+    assert sol_2 == Matrix([[0] , [0] , [0]])
```

Comment:
This issue number doesn't match this code. It should be 19815 and it's the example tested above so that's where the comment should go.